app/serializers/api/v1/serialize_time_value.rb

```diff
@@ -0,0 +1,29 @@
+module Api::V1
+  module SerializeTimeValue
+
+    def self.included(base)
+      base.instance_eval do
+        def self.time_attributes
+          model_name.columns_hash.select{|k,v| v.type == :datetime}.keys
```

Comment:
Unused block argument - `k`. If it's necessary, use `_` or `_k` as an argument name to indicate that it won't be used.<br>Space missing after comma.<br>Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.